msg/versioned/VehicleCommand.msg

```diff
@@ -109,7 +109,7 @@ uint16 VEHICLE_CMD_EXTERNAL_WIND_ESTIMATE = 43004
 
 # PX4 vehicle commands (beyond 16 bit mavlink commands)
 uint32 VEHICLE_CMD_PX4_INTERNAL_START    = 65537        # start of PX4 internal only vehicle commands (> UINT16_MAX)
-uint32 VEHICLE_CMD_SET_GPS_GLOBAL_ORIGIN = 100000       # Sets the GPS coordinates of the vehicle local origin (0,0,0) position. |Empty|Empty|Empty|Empty|Latitude|Longitude|Altitude|
+uint32 VEHICLE_CMD_SET_GPS_GLOBAL_ORIGIN = 100000       # Sets the GPS coordinates of the vehicle local origin (0,0,0) position. |Empty|Empty|Empty|Empty|Latitude (WGS-84)|Longitude (WGS-84)|Altitude (AMSL from GNSS, positive above ground)|
```

Comment:
This OK? FYI this is a bit odd. There is no such command in MAVLink - it's a message.

Reply:
The ID is 48 (<256) which tells you it is a mavlink 1 message created before there were commands. Easy enough to replace with a message "over time" - depends on whether timestamp in message is actually used/needed?

Proposal: https://github.com/mavlink/mavlink/pull/2247

Reply:
Closing this. We'll make it a real command if the command gets through mavlink.